Source/MOLAuthenticatingURLSession.m

```diff
@@ -242,7 +243,18 @@ - (NSURLCredential *)clientCredentialForProtectionSpace:(NSURLProtectionSpace *)
 ///  If the server's certificate chain does not evaluate for any reason, returns nil.
 ///
 - (NSURLCredential *)serverCredentialForProtectionSpace:(NSURLProtectionSpace *)protectionSpace {
-  SecTrustRef serverTrust = protectionSpace.serverTrust;
+  SecTrustRef serverTrust = NULL;
+  CFArrayRef policies = NULL;
```

Comment:
Move lines 247-256 to line 289, just above where the anchor certs are set as we only need to do this if setting the anchor cert. Also, add a comment explaining why this is necessary.